### DISCUSSION TITLE
How to hide the Activity on the UI

### DISCUSSION THREAD

**Discussion by hengshanMWC** [reown-com/appkit#3319] [Q&A]
<img width="638" alt="image" src="https://github.com/user-attachments/assets/2a98eed0-3e51-4a24-8ecc-d0843ff700fa">
<img width="470" alt="image" src="https://github.com/user-attachments/assets/2561b88a-697a-4ef5-a78c-f478617bb78e">

**Comment by arein** [comment]
curious @hengshanMWC - why do you want to hide it?

**Reply by hengshanMWC** [reply]
Because it is not used for the time being, and I don't want to confuse users, can you add a field control? I can also help

**Reply by arein** [reply]
> I can also help

Sure contributions are welcome :)

**Reply by hengshanMWC** [reply]
> > I can also help 我也可以帮忙
> 
> Sure contributions are welcome :)欢迎肯定的贡献:)
Hello, I just thought about it and this function is helpful to me
